WordPress/Classes/ViewRelated/Post/LoadMoreCounter.swift

```diff
@@ -0,0 +1,40 @@
+import Foundation
+
+/// Temporary counter to help investigate an old issue with load more analytics being spammed.
+/// See https://github.com/wordpress-mobile/WordPress-iOS/issues/6819
+class LoadMoreCounter {
+
+    var count: Int = 0
+    // Set dryRun to true to avoid bumping stats during tests.
+    var dryRun = false
+
+    init(startingCount: Int = 0) {
```

Comment:
I like the way you used a default parameter to injected the initial count here! It occurred to me that you could do a similar thing for `dryRun` (e.g., `, isTesting: Bool = false`). This would allow you to make `dryRun` private on line 9.

Reply:
Good call.